src/Microsoft.Identity.Client/Cache/CacheFallbackOperations.cs

```diff
@@ -117,11 +118,13 @@ internal static class CacheFallbackOperations
         /// 
         /// Item1 is a map of ClientInfo -> AdalUserInfo for those users that have ClientInfo 
         /// Item2 is a list of AdalUserInfo for those users that do not have ClientInfo
-        /// </summary>
+        /// </summary>
+        /// <remarks>Users are not filetered by enviroment</remarks>

```

Comment:
>filetered by enviroment [](start = 35, length = 23)

sp: filtered by environment

Reply:
Well actually that comment is old. Users are filtered by env now.